### PR TITLE
AddRedis when using Microsoft.AspNetCore.SignalR.Redis

### LINKED ISSUE #15528: AddRedis when using Microsoft.AspNetCore.SignalR.Redis
>In the Startup.ConfigureServices method, call AddStackExchangeRedis after AddSignalR:

Apparently `AddRedis` is the method when using `Microsoft.AspNetCore.SignalR.Redis`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c548355c-d88f-b941-0622-56732973123b
* Version Independent ID: bddd7575-422d-4342-af48-f6b0979fb69a
* Content: [Redis backplane for ASP.NET Core SignalR scale-out](https://docs.microsoft.com/en-us/aspnet/core/signalr/redis-backplane?view=aspnetcore-3.0)
* Content Source: [aspnetcore/signalr/redis-backplane.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/signalr/redis-backplane.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-signalr**
* GitHub Login: @bradygaster
* Microsoft Alias: **bradyg**

### PR DESCRIPTION
**Pull request by spottedmahn**
Fixes #15528

@Rick-Anderson  edit: 
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/redis-backplane?view=aspnetcore-3.0&branch=pr-en-us-15763)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@BrennanConroy  if you can review the technical changes I'll wordsmith this.

File: aspnetcore/signalr/redis-backplane.md [hunk 1]
```diff
@@ -65,10 +65,46 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
...
+    });
+  ```
+
+  In the preceding code, `options.Configuration` is initialized with whatever was specified in the connection string.
+
+  For information about Redis options, see the [StackExchange Redis documentation](https://stackexchange.github.io/StackExchange.Redis/Configuration.html).
+
+::: moniker-end
+
+::: moniker range="> aspnetcore-3.0"
```
Comment (BrennanConroy) [inline aspnetcore/signalr/redis-backplane.md]: I think this should be >=

File: aspnetcore/signalr/redis-backplane.md [hunk 2]
```diff
@@ -65,10 +65,46 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
...
+    .AddStackExchangeRedis(connectionString, options => {
+        options.Configuration.ChannelPrefix = "MyApp";
+    });
+  ```
+
+  In the preceding code, `options.Configuration` is initialized with whatever was specified in the connection string.
+
+  For information about Redis options, see the [StackExchange Redis documentation](https://stackexchange.github.io/StackExchange.Redis/Configuration.html).
+
+::: moniker-end
```
Comment (BrennanConroy) [inline aspnetcore/signalr/redis-backplane.md]: This whole section will show up twice now because the moniker is `> aspnetcore-2.1`

**Comment by Rick-Anderson** [comment]
@BrennanConroy  I'll fix the monikers and wordsmith.

**Comment by BrennanConroy** [comment]
Ok, content is fine

File: aspnetcore/signalr/redis-backplane.md [hunk 3]
```diff
@@ -56,50 +55,49 @@ This article explains SignalR-specific aspects of setting up a [Redis](https://r
...
+  * `Microsoft.AspNetCore.SignalR.Redis` - Depends on StackExchange.Redis 1.X.X. This package isn't included in ASP.NET Core 3.0 and later.
 
-Note: If using `Microsoft.AspNetCore.SignalR.Redis` then call `AddRedis`
+* In the `Startup.ConfigureServices` method, call <xref:Microsoft.Extensions.DependencyInjection.StackExchangeRedisDependencyInjectionExtensions.AddStackExchangeRedis*>:
 
   ```csharp
   services.AddSignalR().AddStackExchangeRedis("<your_Redis_connection_string>");
   ```
 
+ When using `Microsoft.AspNetCore.SignalR.Redis`, call <xref:Microsoft.Extensions.DependencyInjection.RedisDependencyInjectionExtensions.AddRedis*>.
```
Comment (spottedmahn) [inline aspnetcore/signalr/redis-backplane.md]: neat 👍.  I assume that create some type of hyperlink? 🤔

Comment (Rick-Anderson) [reply aspnetcore/signalr/redis-backplane.md]: Yes

**Pull request merged**